src/rpc/blockchain.cpp

```diff
@@ -1338,6 +1338,17 @@ static UniValue BIP9SoftForkDesc(const Consensus::Params& consensusParams, Conse
     rv.push_back(Pair("startTime", consensusParams.vDeployments[id].nStartTime));
     rv.push_back(Pair("timeout", consensusParams.vDeployments[id].nTimeout));
     rv.push_back(Pair("since", VersionBitsTipStateSinceHeight(consensusParams, id)));
+    if (THRESHOLD_STARTED == thresholdState)
```

Comment:
this duplicates #2435 which should also be (partially?) reverted I guess

Reply:
https://github.com/dashpay/dash/pull/2992/commits/d955e7af261c86dac68267cedf9b6d38f3002180